### PR TITLE
Fixed Issue #30 and #29 : Adding Home button, Updating Footer

### PR DESCRIPTION
**Pull request by arghac14**
@pranaykgupta I have added the button on portal.php and updated the footer.
please check!
Capture
c2

### CODE REVIEW

File: portal.php [hunk 1]
```diff
@@ -115,9 +115,17 @@
                         <li><a href="#blog">Utilities</a></li>
                         <li><a href="#projects">Newsfeed</a></li>
                         <li><a href="#contact">Contact Us</a></li>
+                        <!-- <li><button class="button" style="vertical-align:middle" onclick="location.href = 'index.php';"><span>HOME </span></button>
+                        </li> -->
+                        <li>
+                            <form action="index.php" method="post" id="log">
+                              
+                             <button class="button" style="vertical-align:middle" onclick="location.href = 'index.php';"><span>Home </span></button>  
+                         </form>
```
Comment (pranaykgupta) [inline portal.php]: Missing closing li tag

File: portal.php [hunk 2]
```diff
@@ -129,9 +137,11 @@
             <div class="logo">
                 <img src="img/logo1.png" class="static">
             </div>
+            <br>
             <nav>
                 <ul>
-                    <li>
```
Comment (pranaykgupta) [inline portal.php]: Why have you removed it?

File: portal.php [hunk 3]
```diff
@@ -115,9 +115,17 @@
                         <li><a href="#blog">Utilities</a></li>
                         <li><a href="#projects">Newsfeed</a></li>
                         <li><a href="#contact">Contact Us</a></li>
+                        <!-- <li><button class="button" style="vertical-align:middle" onclick="location.href = 'index.php';"><span>HOME </span></button>
```
Comment (pranaykgupta) [inline portal.php]: Also, remove this comment and unrequired blank lines that you have committed

**Comment by arghac14** [comment]
@pranaykgupta I have made the necessary changes, I think it can be merged now. Please check!

**Pull request merged**